apps/web/src/scenes/ContentPages/ChangelogPage.tsx

```diff
@@ -0,0 +1,97 @@
+import { useEffect } from 'react';
+import styled from 'styled-components';
+
+import breakpoints, { pageGutter } from '~/components/core/breakpoints';
+import Markdown from '~/components/core/Markdown/Markdown';
+import { VStack } from '~/components/core/Spacer/Stack';
+import { BaseM, TitleDiatypeL } from '~/components/core/Text/Text';
+
+export type ChangelogSection = {
+  header: string;
+  summary: string;
+  improvementsAndFixes: string;
+  _id: string;
+};
+
+type Props = {
+  sections: ChangelogSection[];
+};
+
+function dateToSlug(date: string) {
+  return date.replace(/( |, )/g, '-').toLowerCase();
+}
+
+export default function ChangelogPage({ sections }: Props) {
+  useEffect(() => {
+    if (window.location.hash) {
+      document.getElementById(window.location.hash.slice(1))?.scrollIntoView();
+    }
```

Comment:
nit: i think ideally we want to `useRouter` instead of window wherever possible: https://stackoverflow.com/questions/68985511/how-do-we-get-window-location-hash-from-a-next-js-application

Reply:
good call, done!